lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -337,8 +345,8 @@ defmodule Kernel.SpecialForms do
 
   The right side of `.` may be a word starting in upcase, which represents
   an alias, a word starting with lowercase or underscore, any valid language
-  operator or any name wrapped in single- or double-quotes. Those are all valid
-  examples:
+  operator, any name wrapped in single- or double-quotes, or be missing. Those
```

Comment:
It's a bit confusing IMO to say that the right side of `.` can be missing and that basically `foo.` is the applicable anon function that we apply with `()` then. Wdyt? Maybe it's better to say that the right side can also be a call with parens and arguments or something like that? The AST backs you up but it's still a bit confusing to read 😕